modules/compass.md

```diff
@@ -0,0 +1,80 @@
+# Overview
+
+```
+Module Name: Compass Bidder Adapter
+Module Type: Compass Bidder Adapter
+Maintainer: sa-support@brightcom.com
+```
+
+# Description
+
+Connects to Compass exchange for bids.
+
+Compass bid adapter supports Banner, Video (instream and outstream) and Native.
+
+# Test Parameters
```

Comment:
Add another # to this title.

``## Test Parameters instead of # Test Parameters``

Codacy is very picky haha